Sources/TuistGenerator/Generator/InfoPlistContentProvider.swift

```diff
@@ -153,4 +202,10 @@ final class InfoPlistContentProvider: InfoPlistContentProviding {
     fileprivate func extend(_ base: inout [String: Any], with: [String: Any]) {
         with.forEach { base[$0.key] = $0.value }
     }
+
+    private func findHost(for target: Target, in project: Project) -> Target? {
```

Comment:
What about moving this to the graph model? I think this is something that we might need to reuse from some other places.